ConsoleApp2/Snake.cs

```diff
@@ -7,23 +7,29 @@ namespace ConsoleApp2
 {
     class Snake
     {
-        private int SizeX;
-        private int SizeY;
+        private int sizeX;
+        private int sizeY;
         public Snake(int pX, int pY, int ScreenSizeX, int ScreenSizeY)
         {
             PositionX = pX;
             PositionY = pY;
-            SizeX = ScreenSizeX;
-            SizeY = ScreenSizeY;
+            sizeX = ScreenSizeX;
+            sizeY = ScreenSizeY;
         }
 
-        public int PositionX { get; set; }
-        public int PositionY { get; set; }
+        public int PositionX { get; private set; }
+        public int PositionY { get; private set; }
 
         private int speed = 1;
```

Comment:
Mark this field "const" or rewrite field as "property" that has only "get" method.